index.html

```diff
@@ -1170,9 +1151,123 @@
             all remaining steps.
             </li>
             <li>Otherwise, the user <em>granted permission</em> to use a
-            display; let <var>D</var> be that display and <var>U</var> the user
-            agent connected to <var>D</var>.
+            display; let <var>D</var> be that display.
+            </li>
+            <li>Run the steps to <a>start a presentation connection</a> with
+            <var>presentationRequest</var>, <var>D</var>, and <var>P</var>.
+            </li>
+          </ol>
+          <div class="note">
+            The details of implementing the permission request and display
+            selection are left to the user agent; for example it may show the
+            user a dialog and allow the user to select an available display
+            (granting permission), or cancel the selection (denying
+            permission). Implementers are encouraged to show the user whether
+            an available display is currently in use, to facilitate
+            presentations that can make use of multiple displays.
+          </div>
+          <div class="note">
+            Receiving user agents are encouraged to advertise a user friendly
+            name for the presentation display, e.g. &quot;Living Room TV&quot;,
+            to assist the user in selecting the intended display. Implementers
+            of receiving user agents are also encouraged to advertise the
+            locale and intended text direction of the user friendly name.
+            Implementers of controlling user agents are encouraged to render a
+            user friendly name using its locale and text direction when they
+            are known.
+          </div>
+        </section>
+        <section>
+          <h4>
+            Starting a presentation from a default presentation request
+          </h4>
+          <p>
+            When the user expresses an intent to start presentation of a
+            document on a presentation display using the browser chrome (via a
+            dedicated button, user gesture, or other signal), that user agent
+            MUST run the following steps to <dfn data-lt=
+            "starting a presentation from a default presentation request">start
+            a presentation from a default presentation request</dfn>. If no
+            <a>default presentation request</a> is set on the document, these
+            steps MUST not be run.
+          </p>
+          <dl>
+            <dt>
+              Input
+            </dt>
+            <dd>
+              <var>W</var>, the document that the user intends to present
```

Comment:
I may be reading that incorrectly but, for me, the document that the user intends to present is the document that will be created on the receiving side when the presentation gets started. Use "the document on which the user expresses an intent to start presentation", instead?